doc/md/guides/quickstart.mdx

```diff
@@ -114,9 +134,14 @@ Run the rest of the commands in this guide from the root of the repository.
 The Bank of Holos platform is organized as a collection of software components.
```

Comment:
Change to `Configure GitOps` to clarify ArgoCD isn't a mandatory part of the platform